plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-calculate-shipping.spec.js

```diff
@@ -137,12 +137,21 @@ test.describe( 'Cart Block Calculate Shipping', () => {
 		await addAProductToCart( page, product1Id );
 		await page.goto( cartBlockPage.slug );
 
+		await page
+			.locator( 'button', {
+				hasText: 'Enter the address to check delivery options',
+			} )
+			.click();
```

Comment:
Instead of `page.locator()` we should use `page.getByRole()`. Something like this:

```ts
await page.getByRole( 'button', { name: 'Enter the address to check delivery options' } ).click();
```